pytype/directors_test.py

```diff
@@ -305,8 +305,8 @@ def check_warning(message_regex, text):
       self.assertEqual(1, error.lineno)
       self.assertRegex(str(error), message_regex)
 
-    check_warning("Unknown pytype directive.*disalbe.*",
-                  "# pytype: disalbe=test-error")
+    check_warning("Unknown pytype directive.*disable.*",
```

Comment:
I'm pretty sure this is an intentional misspelling - the test is making sure that pytype reports an error when "disable" is spelled wrong.

Reply:
@rchen152 Noted. Changes reverted. Please check again.